src/node/clean-up-temp-files.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+'use strict';
+
+const path = require('path');
+const del = require('del');
+const pTry = require('p-try');
+
+const cleanUpTempFiles = (
+  batfishConfig: BatfishConfiguration
+): Promise<void> => {
+  const assetsDirectory = path.join(
+    batfishConfig.outputDirectory,
+    batfishConfig.publicAssetsPath
+  );
+
+  const assetsJsonPath = path.join(assetsDirectory, 'assets.json');
+  const renderStaticPagesPath = path.join(
+    assetsDirectory,
+    'static-render-pages.js'
+  );
+  const renderStaticPagesMapPath = path.join(
+    assetsDirectory,
+    'static-render-pages.js.map'
+  );
+
+  return pTry(() => {
+    return del(
+      [assetsJsonPath, renderStaticPagesPath, renderStaticPagesMapPath],
+      { force: true }
+    );
+  });
```

Comment:
No need for a `pTry` wrapping the `del` call, since `del` already returns a Promise.